### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by EduMorenoJ**
**Description**
Solve this issue https://github.com/onnx/onnx/issues/4222

Signed-off-by: Edu Moreno [eduardomoreno2594@gmail.com](mailto:eduardomoreno2594@gmail.com)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 numpy >= 1.16.6 # TODO: update once TensorFlow 2.6 is end of life
-protobuf >= 3.12.2
-typing-extensions >= 3.6.2.1
\ No newline at end of file
+protobuf == 3.20.0
```
Comment (mgoin) [inline requirements.txt:2]:
```suggestion
protobuf >= 3.12.2, <= 3.20.1
```

Comment (EduMorenoJ) [reply requirements.txt]: :+1:

**Review by dhilazo — APPROVED** [review]

**Comment by etiotto** [comment]
@jcwchen @gramalingam should we include this PR into v1.12.0 ?

**Comment by jcwchen** [comment]
Yes I think we should. Thank you for catching this!

**Review by jcwchen — APPROVED** [review]

**Pull request merged**